packages/odata-service-inquirer/test/unit/prompts/cap-project/cap-helper.test.ts

```diff
@@ -138,6 +146,58 @@ describe('cap-helper', () => {
         expect(findCapProjectsSpy).toHaveBeenCalledWith({ 'wsFolders': ['/test/mock/'] });
     });
 
+    test('Should use realpath to convert Windows paths to correct casing - getCapProjectChoices', async () => {
+        const findCapProjectsSpy = jest
+            .spyOn(sapuxProjectAccess, 'findCapProjects')
+            .mockResolvedValue(['c:\\test\\mock\\BOOKSHOP', 'c:\\test\\mock\\FLIGHT']);
+        jest.spyOn(sapuxProjectAccess, 'getCapCustomPaths').mockResolvedValue({
+            app: 'app/',
+            db: 'db/',
+            srv: 'srv/'
+        });
+        jest.spyOn(fs.promises, 'realpath').mockImplementation(async (path: PathLike) => {
+            // Simulate realpath converting paths to correct casing
+            if (path === 'C:\\test\\mock\\BOOKSHOP') {
+                return 'c:\\Test\\Mock\\Bookshop';
+            }
+            if (path === 'C:\\test\\mock\\FLIGHT') {
+                return 'c:\\Test\\Mock\\Flight';
+            }
+            return String(path);
+        });
+
+        const choices = await getCapProjectChoices(['C:\\test\\mock\\']);
+        expect(choices).toMatchInlineSnapshot(`
+            [
+              {
+                "name": "c:\\test\\mock\\BOOKSHOP",
```

Comment:
Drive letters have not been changed to the test is invalid.

Reply:
updated the input for `getCapProjectChoices(`) to the upper case `C:`, opposite to the snapshots with lower case `c:`